update articles link between assets & products (#431)
akeneo/pim-helpcenter: content/md/products-super-power/work-on-a-product.md

## Work with assets in an asset collection attribute _(EE only)_

> In the product form, there is a new tab "Assets" in which you will retrieve all the asset collections of your product. It means that the assets will not be mixed with other values of the product. The asset collections also have a brand new design, here is how they look like:

Reviewer: ```suggestion
In the product form, there is a new "Assets" tab in which you will retrieve all the asset collections of your product. It means that the assets will not be mixed with other values of the product. The asset collections also have a brand new design, here is how it looks:
```

![drag_drop](../img/drag_drop.jpg)

If you have one or several asset collection attribute(s), you can now organize your assets directly in the asset collection in the product form with **the arrows**, and find this order in export file. Simple. Fast. Effective.

You need to import assets in a certain order in Akeneo PIM? No problem! Just order your assets in your import file, separating asset names with a comma, then have this order in the PIM. You can also achieve this by using the API.

The order defined in the PIM will also be the same when you'll export your products with the API.

::: tips
And if you want to have a look at history, a new version is created in `History` tab each time the user changes the asset order.
:::

### Assets display

> If you have assets linked to your products, you can have a **large display of those assets** clicking on their thumbnails.

Reviewer 2: Allez plus dans le détail des previews qui sont maintenant possibles

Author: Done

![drag_drop](../img/drag_drop.jpg)

If you have one or several asset collection attribute(s), you can now organize your assets directly in the asset collection in the product form with **the arrows**, and find this order in export file. Simple. Fast. Effective.

> You need to import assets in a certain order in Akeneo PIM? No problem! Just order your assets in your import file, separating asset's names with a coma, then have this order in the PIM. You can also achieve this by using the API.

Reviewer: ```suggestion
You need to import assets in a certain order in Akeneo PIM? No problem! Just order your assets in your import file, separating asset names with a comma, then have this order in the PIM. You can also achieve this by using the API.
```

And moreover, we added a slideshow to navigate between the assets.

Several actions are available in this asset preview screen:

> - You can download each asset clicking on `download`.

Reviewer: ```suggestion
- You can download each asset by clicking on `download`.
```

> - You can also enrich the asset attributes, clicking on `edit asset`. This way, you will be redirected on the asset form.

Reviewer: ```suggestion
- You can also enrich the asset attributes, clicking on `edit asset`. This way, you will be redirected to the asset form.
```

1.  Click on the `Copy` button, the selected values are copied to the Product Form
  ![Copy](../img/Products_CompareTranslateCopy.png)

  ::: info
  Note that this feature is not available on the new asset collection attributes.
  :::

# View the product’s versions

> The history of changes applied to a product are taking into account for the following operations:

Reviewer: ```suggestion
The history of changes applied to a product are taken into account for the following operations:
```
